### ISSUE TITLE
Maximizing TPR on Adult dataset using Keras Model

### ISSUE THREAD

**Issue by hannanabdul55** [google-research/tensorflow_constrained_optimization#8]
I am trying to run [the Recall constraint notebook](https://github.com/google-research/tensorflow_constrained_optimization/blob/master/examples/colab/Recall_constraint_keras.ipynb) using my own Keras model to implement Equality of Opportunity constraint under a sensitive attribute instead of Recall on the Adult Income dataset.  
But using the mentioned `KerasPlaceholder` will only fetch me the current batch predictions and labels. How can I fetch the corresponding sensitive attribute in the input dataset from the `KerasPlaceholder`?  Basically, how do I implement the constraint implemented in [the adult notebook](https://github.com/google-research/tensorflow_constrained_optimization/blob/master/examples/jupyter/Fairness_adult.ipynb) using the `KerasPlaceholder`?  Any help would be much appreciated!  
Thank you.
